src/openzaak/utils/middleware.py

```diff
@@ -15,3 +33,72 @@ def __call__(self, request):
 
     def log(self, request: HttpRequest):
         logger.debug("Request headers for %s: %r", request.path, request.headers)
+
+
+class JWTAuth(_JWTAuth):
```

Comment:
I'd move this to `components.authorizations.middleware` instead of keeping it in `utils`

Reply:
done